inst/apphelp/descriptionCnaTab.md

```diff
@@ -0,0 +1,26 @@
+**Load a valid copy number data file.** This file is used in cBioPortal for the copy number data panel (see image on the right.)
+
+---
+
+#### A minimal copy number data file with only the columns required for cBioPortal:
+
+The discrete copy number data file contain values that would be derived from copy-number analysis algorithms like GISTIC 2.0 or RAE. For each gene (row) in the data file, the following columns are required in the order specified:
```

Comment:
The discrete copy number data file **contains** for each gene-sample combination a copy number level. By default the following applies:
<Here the list>

The user can currently upload the file "all_thresholded.by_genes.txt" derived from GISTIC 2.0 or a tab-separated file containing the following columns: